zerver/migrations/0465_rename_followup_day_email_templates.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 4.2.2 on 2023-07-11 10:45
+
+import orjson
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+WELCOME = 1
```

Comment:
Might be worth a comment where this constant would normally be imported from, so it doesn't seem quite so arbitrary.

Reply:
Added comment in updated migration.